metrics/web/lib/common/presentation/widgets/projects_search_input.dart

```diff
@@ -0,0 +1,73 @@
+// Use of this source code is governed by the Apache License, Version 2.0
```

Comment:
📱 
I believe it would be better to place this widget under the `common/presentation/project_search_input/widgets` package.